roles/example-cnf-app/templates/testpmd-cr.yaml.j2

```
@@ -5,18 +5,33 @@ metadata:
   namespace: {{ cnf_namespace }}
 spec:
   image_testpmd: registry.redhat.io/openshift4/dpdk-base-rhel8:v4.6
-  privileged: false
   imagePullPolicy: {{ image_pull_policy }}
 {% if enable_lb|bool %}
   ethpeerMaclist: {{ lb_cnf_port_mac_list }}
   size: {{ testpmd_app_size|default(2) }}
 {% else %}
   ethpeerMaclist: {{ trex_mac_list }}
-  size: 1  
+  size: 1
 {% endif %}
   networks: {{ cnf_app_networks }}
   terminationGracePeriodSeconds: {{ termination_grace_period_seconds }}
 {% if mac_workaround_enable|default(false)|bool %}
   macWorkaroundEnable: true
   macWorkaroundVersion: "{% if 'v0.0.' in operator_version %}4.4{% else %}{{ ocp_version }}{% endif %}"
 {% endif %}
+{% if testpmd_cpu|default('') %}
+  cpu: {{ testpmd_cpu }}
+{% endif %}
+{% if testpmd_hugepages|default('') %}
+  thugepage_1gb_count: {{ testpmd_hugepages }}
+{% endif %}
+{% if testpmd_forwarding_cores|default('') %}
+  environments:
+    forwardingCores: {{ testpmd_forwarding_cores }}
+{% endif %}
+{% if testpmd_privileged|default(false)|bool %}
+  privileged: {{ testpmd_privileged }}
+{% endif %}
```

I think you can remove the conditional for any boolean and always be explicit of its value, e.g.

```suggestion
  privileged: {{ testpmd_privileged|default(false)|bool }}
```